src/util/truffleConfig.js

```diff
@@ -120,8 +127,8 @@ function truffleConfig ({
           optimizer: {
             enabled: optimizedEnabled, // Default: false
             runs: 200
-          }
-          // evmVersion: "byzantium"  // Default: "byzantium". Others:  "homestead", ...
+          },
+          evmVersion: "byzantium"  // Default: "byzantium". Others:  "homestead", ...
```

Comment:
Is this really needed. Aren't we on Constantinople?

Reply:
This is probably a copy paste mistake (since we have it in out truffle config).